perfTestUtils/dataStructures.go

```diff
@@ -182,17 +182,25 @@ type PerfStats struct {
 	PeakMemory           uint64
 	ServiceResponseTimes map[string]int64
 	ServiceTransCount    map[string]*uint64
+	ServiceErrorCount    map[string]*uint64
 	ServiceTPS           map[string]float64
 	OverAllTransCount    uint64
+	OverAllErrorCount    uint64
 	OverAllTPS           float64
 	MemoryAudit          []uint64
 	TestPartitions       []TestPartition
-	TestDate             time.Time
+	TestTimeStart        time.Time
+	TestTimeEnd          time.Time
 }
 
```

Comment:
Added GetTestTimeEnd() to format end time on the report. Renamed GetTestTime() function for consistency.